app/uk/gov/hmrc/exports/models/declaration/notifications/Notification.scala

```diff
@@ -16,20 +16,36 @@
 
 package uk.gov.hmrc.exports.models.declaration.notifications
 
-import java.time.LocalDateTime
+import java.time.{Instant, LocalDateTime, ZoneOffset}
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Json, Reads, _}
 import uk.gov.hmrc.exports.models.declaration.submissions.SubmissionStatus.SubmissionStatus
 
 case class Notification(
   actionId: String,
   mrn: String,
-  dateTimeIssued: LocalDateTime,
+  dateTimeIssued: Instant,
   status: SubmissionStatus,
   errors: Seq[NotificationError],
   payload: String
 )
 
 object Notification {
-  implicit val format: OFormat[Notification] = Json.format[Notification]
+  implicit val readLocalDateTimeFromString: Reads[Instant] = implicitly[Reads[LocalDateTime]]
```

Comment:
I'm wondering if it's worth to extract this to some helper object. It's used in two places.

Additionally we can add documentation note with information why this etc. For future reference

Reply:
The comments are on the commit messages :P, where they should live.
I actually prefer it in the code, as it is concise and easy to see.